install.md

```diff
@@ -138,6 +138,26 @@ sudo apt-get install \
   uidmap
 ```
 
+On Manjaro (and maybe other Linux distributions):
+
+Make sure that the Linux kernel supports user namespaces:
+
+```
+> zgrep CONFIG_USER_NS /proc/config.gz
+CONFIG_USER_NS=y
+
+```
+
+If not, please update the kernel.
+For Manjaro Linux the instructions can be found here:
+https://wiki.manjaro.org/index.php/Manjaro_Kernels
+
+After that enable user namespaces:
+
+```
+sudo sysctl kernel.unprivileged_userns_clone=1
```

Comment:
Is there a way to make this permanent?  IE Drop a file into /etc/sysctl.d?

Reply:
I added the instruction to make it permanently. I verified it on my Manjaro Linux machine.